core/flyout.js

```diff
@@ -798,6 +799,7 @@ Blockly.Flyout.prototype.show = function(xmlList) {
   } else {
     this.width_ = 0;
   }
+  this.workspace_.setResizesEnabled(true);
```

Comment:
Maybe put a note or add to the bug that we should consider putting this after reflow? Reflow calls targetWorkspace.resize at the end, so theoretically we could skip that one in favor of the one triggered when you call setResizesEnabled(true) (i.e. what you were telling me yesterday).